src/nodes/Table/TableView.vue

```diff
@@ -84,10 +140,41 @@ table {
 	opacity: .5;
 	position: absolute;
 	top: 0;
-	right: 3px;
+	right: var(--default-clickable-area);
+
+	&:hover {
+		opacity: 1;
+	}
+}
+
+.table-add-column {
+	visibility: hidden;
+	padding-left: 3px;
+	opacity: .5;
+	position: absolute;
+	top: var(--default-clickable-area);
+	right: 0;
+	bottom: 60px;
+	margin-top: 0 !important;
+
+	&:hover {
+		opacity: 1;
+		visibility: unset;
```

Comment:
In my understanding the `visibility: unset` is superfluous. The button already becomes visible on table focus, and it cannot be hovered when `invisible: hidden` anyway.